docs/api/graphql.md

```diff
@@ -0,0 +1,54 @@
+# GraphQL
+
+[GraphQL](https://graphql.org/) is a query language for the API.
+The GraphQL implementation for eZ Platform is located in [`ezsystems/ezplatform-graphql`](https://github.com/ezsystems/ezplatform-graphql).
+
+## Setup
+
+To use GraphQL, you need to generate the domain schema:
+
+``` bash
+php bin/console ezplatform:graphql:generate-schema
+php bin/console cache:clear
+```
+
+This produces YAML files located in `app/config/graphql/ezplatform`.
+They contain the information about the domain objects and the fields you can query.
+
+Every time you modify Content Types in your installation, you need to regenerate the schema
+using the command above.
+
+## Domain schema
+
+GraphQL for eZ Platform is based on the Content Types, Content Type groups and Content items
+defined in the repository.
+
+For each Content Type the schema exposes a singular and plural field, e.g. `article` and `articles`.
+Use the singular field to query a single Content item, and the plural to get a whole `Connection`,
```

Comment:
Collection i guess?

Reply:
Actually it is Connection, it's a GraphQL term (cf. https://github.com/ezsystems/ezplatform-graphql/blob/master/src/GraphQL/Relay/DomainConnectionBuilder.php or https://blog.apollographql.com/explaining-graphql-connections-c48b7c3d6976)

Reply:
It is Connection indeed. It may be worth explaining a bit, like "a Connection, a collection of items that supports pagination" ?